homework1/src/ReplaceElements.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Arrays;
+
+public class ReplaceElements {
+    public static void main(String[] args) {
+        Integer[] numbers = {7, 10, 11, 15, 2, 3, 3, 10, 11, 20};
+        int numbers_length = numbers.length / 2;
```

Comment:
that's not numbers_length, that naming can lead to misunderstanding in more complex cases

Reply:
Fixed in commit: f1536a4